server/engine/src/main/java/org/eclipse/lsp/cobol/core/preprocessor/delegates/writer/CobolLineWriter.java

```diff
@@ -59,7 +59,7 @@ public ExtendedDocument serialize(List<CobolLine> lines, String documentUri) {
       if (!isContinuationLine) {
         if (start != null) {
           Position stop =
-                  new Position(line.getNumber() - 1, sb.length() - sb.lastIndexOf("\n") - 1);
+                  new Position(line.getNumber() - 1, sb.length() - sb.lastIndexOf("\n"));
```

Comment:
I think that the `-1` should have been kept in the code. This was computing the length of the last line (so excluding the newline on the previous line).